ftw/solr/browser/configure.zcml

```diff
@@ -1,8 +1,15 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
     xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:theme="http://namespaces.zope.org/plonetheme.onegov"
```

Comment:
Took it from here: https://github.com/OneGov/plonetheme.onegov/blob/master/plonetheme/onegov/configure.zcml#L9
Should it be `xmlns:theme="http://namespaces.zope.org/theme"`? Is this even used somewhere?